proto/v1beta1/swagger/grafeas.swagger.json

```diff
@@ -15,13 +15,13 @@
     "application/json"
   ],
   "paths": {
-    "/v1beta1/{name}": {
+    "/v1beta1/{name=projects/*/notes/*}": {
```

Comment:
Was this auto-generated or manually added? I don't believe this path would actually be interpreted correctly by swagger because OpenAPI doesn't support paths with wildcards in them.

Reply:
These are likely caused by a new version of protoc that we've pulled in because I don't think we lock in a specific version of protoc to use. So these changes should be fine.